extensions/amp-list/0.1/amp-list.js

```diff
@@ -204,6 +208,21 @@ export class AmpList extends AMP.BaseElement {
       return this.scheduleRender_(items);
     }, error => {
       throw user().createError('Error fetching amp-list', error);
+    }).then(() => {
+      if (this.getFallback()) {
+        // Hide in case fallback was displayed for a previous fetch.
+        this.toggleFallbackInMutate_(false);
+      }
+      this.togglePlaceholder(false);
```

Comment:
this is a change from previous code. These should happen automatically anyway. I recommend keeping the old code structure to eliminate any regressions e.g.
```
const fetch =...
fetch.then() {
 if(fallback) etc...
}
return fetch;
```

Reply:
You mean move this `.then(...)` block back to `layoutCallback()`? That doesn't work because then the loading and placeholder don't go away after a reload (either from the refresh action or an amp-bind change). I just tried it now.

Reply:
I see